### PR TITLE
Port improvements for Magento2 driver from Valet Plus project

### PR DESCRIPTION
**Pull request by lazyguru**
This ports improvements for the Magento2 driver from the Valet Plus project (https://github.com/weprovide/valet-plus) back to this project (which is sort of an upstream)

This fixes an issue with Magento 2.2 that was reported to me outside of this repo by @piotrekkaminski (Product Manager at Magento)

### CODE REVIEW

File: cli/templates/magento2.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<html>
```
Comment (timneutkens) [inline cli/templates/magento2.php]: This can be removed

Comment (lazyguru) [reply cli/templates/magento2.php]: Actually it’s not Valet+ specific. It’s used on line 113 of the driver

Comment (timneutkens) [reply cli/templates/magento2.php]: Ah alright 👍 🙌

Comment (drbyte) [reply cli/templates/magento2.php]: ... but is it really needed? A simple `echo` could accomplish the same purpose.

Comment (lazyguru) [reply cli/templates/magento2.php]: My intent was to keep it as close as possible to the downstream version so any future improvements from either side would be easy to merge. Plus I like the display of the page with a clickable link to start the web installer.

**Comment by mattstauffer** [comment]
@lazyguru Can you describe in simple terms what these improvements are? We definitely want to merge in fixes, but we don't necessarily have the same perspective on how many features to add, so we want to be able to parse out the differences and merge or not merge them each separately.

Thanks!

**Comment by lazyguru** [comment]
Actually @mattstauffer, I think I'm going to withdraw this PR and suggest people use valet-plus if they need this.

**Comment by mattstauffer** [comment]
@lazyguru 👍

**Pull request closed without merging**